app/Models/SnipeSCIMConfig.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Helper;
+use ArieTimmerman\Laravel\SCIMServer\SCIM\Schema;
+use ArieTimmerman\Laravel\SCIMServer\Attribute\AttributeMapping;
+
+
+class SnipeSCIMConfig extends \ArieTimmerman\Laravel\SCIMServer\SCIMConfig
+{
+    public function __constructor(...$params)
+    {
+        \Log::info("CONSTRUCTOR CALLED!!@!!!! FOr my config thing");
+        self::super(...$params);
+    }
+
+    public function getUserConfig()
+    {
+        // Much of this is copied verbatim from the library, then adjusted for our needs
+        return [
+                
+            // Set to 'null' to make use of auth.providers.users.model (App\User::class)
+            'class' => User::class,
+            
+            'validations' => [
+    
+                'urn:ietf:params:scim:schemas:core:2.0:User:userName' => 'required',
+                'urn:ietf:params:scim:schemas:core:2.0:User:password' => 'nullable',
+                'urn:ietf:params:scim:schemas:core:2.0:User:active' => 'boolean',
+                'urn:ietf:params:scim:schemas:core:2.0:User:emails' => 'required|array',
+                'urn:ietf:params:scim:schemas:core:2.0:User:emails.*.value' => 'required|email',
+                'urn:ietf:params:scim:schemas:core:2.0:User:roles' => 'nullable|array',
+                'urn:ietf:params:scim:schemas:core:2.0:User:roles.*.value' => 'required',
+    
+            ],
+    
+            'singular' => 'User',
+            'schema' => [Schema::SCHEMA_USER],
+    
+            //eager loading
+            'withRelations' => [],
```

Comment:
Do you know what this does? We generally do load users with relations, so I’m not sure what impact this might have.

Reply:
Nope!

Reply:
Definitely not the answer I was hoping for, but I do appreciate your honesty lol